HW04/aleksei_zhuchkov/spec/mentor_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../lib/mentor'
+require_relative '../lib/notification'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'student_name', surname: 'student_surname') }
+  let(:mentor) { described_class.new(name: 'mentor_name', surname: 'mentor_surname') }
+  let(:homework) do
+    mentor.add_homework(title: 'test_title',
+                        description: 'test_description',
+                        student: student)
+  end
+
+  describe 'check method add_homework' do
```

Comment:
please, use method names everywhere

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/3483c4bde37292ce6fb967100b433f4ba9e659f6